Fabric.Identity.API/scripts/identity-cli.psm1

```diff
@@ -27,8 +27,10 @@ function Get-AccessToken {
         [Parameter(Mandatory=$True)] [string] $secret,
         [string[]] $scope
     )
+    
+    $url = ""
+    [System.Uri]::TryCreate($identityUrl, "connect/token", [ref]$url) | Out-Null
```

Comment:
Do we want to do a TryCreate here? Should we just use the Uri constructor that takes a string and let that throw an exception if the Uri is in an invalid format?

Reply:
Yes, this doesn't work as expected, this was a workaround for running against the docker identity, which breaks against a real url.  The issue there was that a trailing slash was being appended regardless, which was causing issues, going to work around it by pulling out the string from the uri and using that instead